Templates/Ros2FleetRobotTemplate/README.md

```diff
@@ -15,30 +15,109 @@ If your simulation does not match this and you would like to start with a simple
 Please follow the instructions in [ROS 2 Gem documentation](https://development--o3deorg.netlify.app/docs/user-guide/interactivity/robotics/project-configuration/)
 to install all required dependencies and create your project with a template (make sure to use chose this template during the process).
 
-## Spawning robots
+## Fleet navigation
 
-The level contains spawn points configured to easily add more Proteus robots through ROS 2 calls.
-This is done with the [Spawner Component](https://development--o3deorg.netlify.app/docs/user-guide/interactivity/robotics/concepts-and-components-overview/#spawner).
-There are 4 spawn points already added in the level. You can use them all with the following service calls:
+This template comes with the example fleet navigation ROS 2 package called `o3de_fleet_nav`. You can find a prepared ROS 2 workspace in the `Examples` directory.
 
-```shell
-ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'proteus', xml: 'spawnPoint1'}'& \
-ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'proteus', xml: 'spawnPoint2'}'& \
-ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'proteus', xml: 'spawnPoint3'}'& \
-ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'proteus', xml: 'spawnPoint4'}'
+This package contains a modified code from `nav2_bringup` package (https://github.com/ros-planning/navigation2).
+
+In this example, a fleet of robots is automatically spawned and each individual robot can be controlled via the Rviz2. An AMCL localization is used for robot localization.
+
+### Fleet configuration
+
+You can configure the fleet by modifying `Example/ros2_ws/src/o3de_fleet_nav/config/fleet_config.yaml` file:
+
+```
+fleet:
+  - robot_name: proteus
+    robot_namespace: robot1
+    position: 
+      x: -6.0
+      y: 0.5
+      z: 0.2
+  - robot_name: proteus
+    robot_namespace: robot2
+    position: 
+      x: -6.0
+      y: 7.5
+      z: 0.2
+  - robot_name: proteus
+    robot_namespace: robot3
+    position: 
+      x: -6.0
+      y: -6.0
+      z: 0.2
 ```
 
-## Topics and frames
+This configuration file contains the data about each robot in a fleet:
+- name (a type of the robot to spawn), 
+- namespace (must be unique per spawned robot),
+- spawning position (spawn position is also used as an AMCL initial estimation). 
+
+In this example, only the `proteus` robot is supported.
+
+You can modify contents of this file to add/remove robots or change their initial positions.
+
+> Notice: You have to rebuild the ROS 2 workspace for changes to update.
+
```

Comment:
One of the most important use cases would be importing own robot and setting it with navigation stack. I believe instructions should provide helpful tips on what needs to be adjusted in navigation stack parameters in order for the new robot to work. Of course this has to be on a general level, and sending users to external documentation resources.

Reply:
Added a section about preparing robot :+1: